properties/src/test/java/io/quarkus/ts/properties/consul/OpenShiftConsulConfigSourceIT.java

```diff
@@ -1,10 +1,10 @@
 package io.quarkus.ts.properties.consul;
 
-import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.condition.DisabledIfSystemProperty;
 
 import io.quarkus.test.scenarios.OpenShiftScenario;
 
 @OpenShiftScenario
-@Disabled("https://github.com/hashicorp/consul/issues/21762")
+@DisabledIfSystemProperty(named = "ts.arm.missing.services.excludes", matches = "true", disabledReason = "Consul for openshift don't have Arm64 image")
```

Comment:
I think the P/Z disable will need to be returned too.

Btw: https://catalog.redhat.com/en/software/containers/hashicorp/consul/612d01d49f14588c41ebf67c?image=68d28f659f56d566a0dd4e12

If I check this, the 1.21.5-ubi release has aarch64 release - can we use this instead?

Reply:
Oh, you are correct, sorry for the noise